src/Components/CollapsiblePanel/CollapsiblePanel.js

```diff
@@ -40,7 +40,7 @@ const propTypes = {
 function CollapsiblePanel(props) {
   const {
     children,
-    className,
+    className = 'neutral-600 bg-white bg-blue-lighter-hover bb b-neutral-300',
```

Comment:
I feel like some of these should always be included, like the hover. It might be better to expand the styles to be more like the Alert where there are different types that have their own associated styles. Otherwise, passing in a styles field is preferable.